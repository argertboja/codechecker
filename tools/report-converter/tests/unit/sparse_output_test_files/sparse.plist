<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>diagnostics</key>
	<array>
		<dict>
			<key>category</key>
			<string>HIGH</string>
			<key>check_name</key>
			<string>sparse</string>
			<key>description</key>
			<string>symbol 'n' was not declared. Should it be static?</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>2363ff8d1b0b7eeffaa0ec3b34679bd9</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>17</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>5</integer>
			</dict>
			<key>path</key>
			<array>
			</array>
			<key>type</key>
			<string>sparse</string>
		</dict>
<dict>
			<key>category</key>
			<string>HIGH</string>
			<key>check_name</key>
			<string>sparse</string>
			<key>description</key>
			<string>sincorrect type in argument 2 (different address spaces) </string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>2383ff8d8b0b7e5ffaa0fc3b34679b89</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>5</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>7</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>5</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>7</integer>
					</dict>
					<key>message</key>
					<string>uexpected long int </string>
				</dict>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>5</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>7</integer>
					</dict>
					<key>message</key>
					<string>got int</string>
				</dict>
			</array>
			<key>type</key>
			<string>sparse</string>
		</dict>
	</array>
	<key>files</key>
	<array>
		<string>files/main.c</string>
	</array>
	<key>metadata</key>
	<dict>
		<key>analyzer</key>
		<dict>
			<key>name</key>
			<string>sparse</string>
		</dict>
		<key>generated_by</key>
		<dict>
			<key>name</key>
			<string>report-converter</string>
			<key>version</key>
			<string>x.y.z</string>
		</dict>
	</dict>
</dict>
</plist>
