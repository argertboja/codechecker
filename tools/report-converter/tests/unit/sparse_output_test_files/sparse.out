main.c:7:5: warning: incorrect type in argument 2 (different address spaces)
main.c:7:5:    expected long int
main.c:7:5:    got int
main.c:5:17: warning: symbol 'n' was not declared. Should it be static?


